### PR TITLE
New build approach using @ima/plugin-cli

### PR DESCRIPTION
**Pull request by jsimck**

### CODE REVIEW

File: createImaPluginConfig.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
+/* eslint-disable jsdoc/require-jsdoc */
```
Comment (Filipoliko) [inline createImaPluginConfig.js:1]: Maybe these default configurations could be part of the `ima-plugin-cli`, since we will be re-using them in different repositories

Comment (jsimck) [reply createImaPluginConfig.js:1]: Yup totally agree, I would probably include them with the migration to `@ima` monorepo if it's ok?

Comment (jsimck) [reply createImaPluginConfig.js]: Fixed with new plugin version

File: jest.config.base.js [hunk 2]
```diff
@@ -3,8 +3,9 @@ module.exports = {
   verbose: true,
   testRegex: '(/src(/?[^/]*){0,5}/__tests__/).*Spec\\.jsx?$',
   modulePaths: ['<rootDir>/'],
+  transformIgnorePatterns: ['node_modules/(?!(@ima/core)/)'],
```
Comment (Filipoliko) [inline jest.config.base.js:6]: Out of curiosity, why is this needed?

Comment (jsimck) [reply jest.config.base.js:6]: The @ima/core is now pure esm package, so it needs to be transformed to cjs with other source files otherwise it fails on imports from @ima/core in Spec files.

File: package.json [hunk 3]
```diff
@@ -27,10 +27,10 @@
...
-    "@ima/cli": "18.0.0-rc.7",
-    "@ima/core": "18.0.0-rc.7",
+    "@ima/cli": "18.0.0-rc.8",
+    "@ima/core": "18.0.0-rc.8",
     "@ima/dev-utils": "^18.0.0-rc.7",
-    "@ima/plugin-cli": "^18.0.0-rc.2",
+    "@ima/plugin-cli": "^18.0.0-rc.3",
     "@ima/server": "18.0.0-rc.7",
     "@types/jest": "^26.0.24",
     "@types/node": "^18.7.6",
```
[the remark this answers is not in the document]
Comment (jsimck) [reply package.json:36]: I've tried this, but figured out that esm in jest is not as simple as renaming these files. 

CLI plugin is currently transpiled from TS to CJS, so it would not work too. 

I would probably leave this to some other PR and not complicate this any more. Also CLI plugins needs to be updated to work with ESM (which will probably happen)

File: .changeset/dry-buttons-reflect.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+---
+"babel-plugin-ima-logger": patch
```
[the remark this answers is not in the document]
Comment (jsimck) [reply .changeset/dry-buttons-reflect.md]: Fixed

**Review by Filipoliko — APPROVED** [review]
